simulator-core/src/test/java/com/evbox/everon/ocpp/functional/availability/HeartbeatTest.java

```diff
@@ -0,0 +1,55 @@
+package com.evbox.everon.ocpp.functional.availability;
+
+import com.evbox.everon.ocpp.testutils.ocpp.exchange.Heartbeat;
+import com.evbox.everon.ocpp.testutils.station.StationSimulatorSetUp;
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+import static com.evbox.everon.ocpp.testutils.constants.StationConstants.STATION_ID;
+import static com.evbox.everon.ocpp.testutils.expect.ExpectedCount.atLeastOnce;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+public class HeartbeatTest extends StationSimulatorSetUp {
```

Comment:
why this test is part of availability package?

Reply:
Availability - G02 - Heartbeat